### PR TITLE
Add obssh data support to WebHMI

### PR DESCRIPTION
**Pull request by YarikMamykin**
Implements/Fixes F132-RC-OBSSH

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
1. Added OBSSH related model and view.
2. Updated settings view and index.html with OBSSH data.
3. Implemented logic for OBSSH feature.

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/controller/SettingsController.js [hunk 1]
```diff
@@ -593,6 +593,14 @@ SDL.SettingsController = Em.Object.create(
         }
     },
 
+    /**
+     * @function turnOnTrailerCheck
```
Comment (mkorniichuk) [inline app/controller/SettingsController.js:597]: @YarikMamykin please, fix docstring

File: app/controller/sdl/RCModulesController.js [hunk 2]
```diff
@@ -42,7 +42,8 @@ SDL.RCModulesController = Em.Object.create({
       CLIMATE: 'climateModels',
       SEAT: 'seatModels',
       LIGHT: 'lightModels',
-      HMI_SETTINGS: 'hmiSettingsModels'
+      HMI_SETTINGS: 'hmiSettingsModels',
+      OBSSH: 'obssh'
```
Comment (mkorniichuk) [inline app/controller/sdl/RCModulesController.js:46]: @YarikMamykin why not `obsshModels`?

File: app/controller/sdl/RCModulesController.js [hunk 3]
```diff
@@ -847,6 +856,21 @@ SDL.RCModulesController = Em.Object.create({
                     dataToReturn.seatControlData = seatControlData;
                 }
             }
+          case 'OBSSH':
```
Comment (mkorniichuk) [inline app/controller/sdl/RCModulesController.js:859]: @YarikMamykin please, fix indentation

File: app/controller/sdl/RCModulesController.js [hunk 4]
```diff
@@ -847,6 +856,21 @@ SDL.RCModulesController = Em.Object.create({
...
+
+                  if(obsshControlData) {
+                    if (Object.keys(obsshControlData).length > 0) {
+                      FFW.RC.onInteriorVehicleDataNotification({moduleType:'OBSSH', moduleId: moduleUUId,
+                        obsshControlData: obsshControlData});
+                    }
+                    dataToReturn.obsshControlData = obsshControlData;
+                  }
+                }
+              break;
```
Comment (mkorniichuk) [inline app/controller/sdl/RCModulesController.js:872]: @YarikMamykin I believe this break is not necessary, but in `case 'SEAT'` one should be added.

File: app/StateManager.js [hunk 5]
```diff
@@ -251,6 +251,13 @@ var StateManager = Em.StateManager.extend(
         }),
 
         seat: Em.State.create({}),
+        obssh: Em.State.create({
```
Comment (mkorniichuk) [inline app/StateManager.js:254]: @YarikMamykin just a suggestion to keep consistency: either add here an empty line or remove in other places.

File: app/view/settings/helpers/OBSSHViewHelper.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+ * @name ObsshViewHelper 
+ * @desc Obssh visual helper
+ * @category View
+ * @filesource app/view/settings/helpers/ObsshViewHelper.js
+ * @version 1.0
+ */
+
+SDL.ObsshViewHelper = Em.Object.create({
+    /**
+     * @function ContainerView
```
Comment (mkorniichuk) [inline app/view/settings/helpers/OBSSHViewHelper.js:37]: @YarikMamykin fix docstring

**Comment by YarikMamykin** [comment]
PR is closed in favor of https://github.ford.com/SmartDeviceLinkMirror/sdl_hmi/pull/72

**Pull request closed without merging**